recipe/meta.yaml

```
@@ -12,28 +14,25 @@ source:
     - setup.py.patch
 
 build:
-  number: 1
+  number: 2
   skip: true  # [not linux]
-  skip: true  # [cuda_compiler_version == "None"]
-  skip: true  # [cuda_compiler_version == "10.2"]
-  skip: true  # [cuda_compiler_version == "11.0"]
+  skip: true  # [cuda_compiler_version != "11.6"]
```

Can generalize the `skip` here to avoid hardcoding a version

Also since the CUDA compiler `run_exports` are incorrect for this case, would suggest we add this to ignore them

```suggestion
  skip: true  # [(not linux) or cuda_compiler_version in (undefined, "None")]
  ignore_run_exports_from:
    - {{ compiler("cuda") }}
```